Analysis/src/TauUncertainties.cpp

```diff
@@ -107,90 +106,78 @@ double TauESUncertainties::GetCorrectionFactorTrueTau(analysis::Period period, i
 {
-    // Values taken from: https://indico.cern.ch/event/865792/contributions/3659828/attachments/1954858/3246751/ETauFR-update2Dec.pdf#page=40
+    // Values taken from: https://indico.cern.ch/event/868279/contributions/3665970/attachments/1959265/3264468/FES_9Dec_explained.pdf
     // For eta < 1.448
-    static const std::map<analysis::Period, std::map<DiscriminatorWP, PhysicalValue>> deep_tau_vs_e_energy_scale_eta_low = {
-      { analysis::Period::Run2016, { { DiscriminatorWP::VVVLoose,  PhysicalValue(0.0,0.0)},
-                                     { DiscriminatorWP::VVLoose,  PhysicalValue(0.020,0.003)},
-                                     { DiscriminatorWP::VLoose,  PhysicalValue(0.030,0.003) },
-                                     { DiscriminatorWP::Loose,  PhysicalValue(0.033,0.005) },
-                                     { DiscriminatorWP::Medium,  PhysicalValue(0.020,0.015) },
-                                     { DiscriminatorWP::Tight,  PhysicalValue(0.06,0.04) },
-                                     { DiscriminatorWP::VTight,  PhysicalValue(0.10,0.06) },
-                                     { DiscriminatorWP::VVTight,  PhysicalValue(0.03,0.10)}} },
-      { analysis::Period::Run2017, { { DiscriminatorWP::VVVLoose,  PhysicalValue(0.0,0.0)},
-                                     { DiscriminatorWP::VVLoose,  PhysicalValue(0.025,0.003)},
-                                     { DiscriminatorWP::VLoose,  PhysicalValue(0.035,0.003) },
-                                     { DiscriminatorWP::Loose,  PhysicalValue(0.038,0.008) },
-                                     { DiscriminatorWP::Medium,  PhysicalValue(0.015,0.018) },
-                                     { DiscriminatorWP::Tight,  PhysicalValue(0.04,0.05) },
-                                     { DiscriminatorWP::VTight,  PhysicalValue(0.04,0.09) },
-                                     { DiscriminatorWP::VVTight,  PhysicalValue(0.07,0.19)}} },
-      { analysis::Period::Run2018, { { DiscriminatorWP::VVVLoose,  PhysicalValue(0.0,0.0)},
-                                     { DiscriminatorWP::VVLoose,  PhysicalValue(0.028,0.003)},
-                                     { DiscriminatorWP::VLoose,  PhysicalValue(0.040,0.003) },
-                                     { DiscriminatorWP::Loose,  PhysicalValue(0.048,0.005) },
-                                     { DiscriminatorWP::Medium,  PhysicalValue(0.035,0.008) },
-                                     { DiscriminatorWP::Tight,  PhysicalValue(0.04,0.03) },
-                                     { DiscriminatorWP::VTight,  PhysicalValue(0.05,0.03) },
-                                     { DiscriminatorWP::VVTight,  PhysicalValue(0.06,0.06)}} }
+    static const std::map<analysis::Period, std::map<int, analysis::StVariable>> deep_tau_vs_e_energy_scale_barrel = {
+      { analysis::Period::Run2016, { {0,  StVariable(0.881, 0.070, 0.066)},
```

Comment:
values do not correspond the ones on page 29. Please cross-check. https://indico.cern.ch/event/868279/contributions/3665970/attachments/1959265/3266335/FES_9Dec_explained.pdf#page=29